dev/envvars.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export CORECLR_ENABLE_PROFILING="1"
+export CORECLR_PROFILER="{918728DD-259F-4A6A-AC2B-B85E1B658318}"
+export CORECLR_PROFILER_PATH="${PWD}/src/Datadog.Trace.ClrProfiler.Native/bin/Debug/x64/OpenTelemetry.AutoInstrumentation.ClrProfiler.Native.so"
```

Comment:
PWD here can be a bit misleading as you assume this will be run from parent directory

Reply:
true. I can try to find some magic script to handle it that supports bash and zsh (AFAIK it is not simple) 😄